src/global/global_init.cc

```diff
@@ -117,9 +117,9 @@ void global_pre_init(
     &cluster, &conf_file_list);
 
   CephContext *cct = common_preinit(iparams, code_env, flags);
-  cct->_conf->cluster = cluster;
-  global_init_set_globals(cct);
   auto& conf = cct->_conf;
+  conf->cluster = cluster;
+  global_init_set_globals(cct);
```

Comment:
this change is confusing. i think the existing code can be interpreted like:

1. update `cct` with values from all sources, like env variables, options, and command line options, and function parameters
1. update the global instances with `cct`
1. create an reference of `cct->_conf`, as it'd be handy to use it later on.

Reply:
also, IMHO, "nit" is probably not the best word to describe an improvement. reviewers use "nits" to reference trivial issues. "minor improvement" or "transpose couple statements" might be better.

Reply:
@tchaikov I'll abandon this commit